docs/internals/howto-release-django.txt

```diff
@@ -83,7 +83,7 @@ permissions.
     ``you@example.com`` is the email address associated with the key you want
     to use.
 
-* A clean Python virtual environment per Django version being released, with
+* A clean Python virtual environment (Python 3.9+) to build artifacts, with
```

Comment:
To build artifacts, we need to use Python 3.9+ so setuptools >= 75.8.1 is used, ensuring proper artifact names (lowercase `d` in `django`) for both the tarball and wheel. This will also be the case when ticket-35961 gets worked on/solved, since the new `project.license` and `project.license-files` are available on setuptools>=77.0.0 which supports Python 3.9+.